dataverse-webapp/src/test/resources/json/JsonDatasetVersion.txt

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
I don't think txt file should be in json folder :P but since it is json, the filename extension should be changed to .json.

Reply:
It turns out there already was a copy of the file in `src/test/resources/txt/util/`. I removed the one I added unnecessarily.